opm/grid/cpgrid/CpGridData.cpp

```diff
@@ -1859,6 +1859,49 @@ CpGridData::getPatchesCells(const std::vector<std::array<int,3>>& startIJK_vec,
     return all_cells;
 }
 
+bool CpGridData::hasNNCs(const std::array<int,3>& startIJK, const std::array<int,3>& endIJK) const
+{
+    bool noNNC = true;
+    /// PATCH CELLS
+    const auto& patch_cells = CpGridData::getPatchCells(startIJK, endIJK);
+    for (const auto cellIdx : patch_cells)
+    {
+        Dune::cpgrid::Entity<0> entity(*this, cellIdx, true);
+        auto cellFaces = this->cell_to_face_[entity];
+        for (const auto face : cellFaces)
+        {
+            Dune::cpgrid::Entity<1> f(face.index(), true);
+            enum face_tag tag = this->face_tag_[f];
+            switch(tag)
+            {
+            case I_FACE:
+                // left / right
+            case J_FACE:
+                // front / back
+            case K_FACE:
+                // bottom / top
+            case NNC_FACE:
+                noNNC = false;
+                OPM_THROW(std::logic_error, "NNC face on a patch cell. Not supported yet!");
```

Comment:
I think would be clearer to the user something like: "NNC face on a cell containing LGR is not supported yet."

Reply:
Indeed, 'patch' is not clear. Thanks!